libp2p/Session.cpp

```diff
@@ -139,43 +123,44 @@ bool Session::readPacket(uint16_t _capId, PacketType _packetType, RLP const& _r)
     return true;
 }
 
-bool Session::interpret(PacketType _t, RLP const& _r)
+bool Session::interpret(P2pPacketType _t, RLP const& _r)
 {
     switch (_t)
     {
     case DisconnectPacket:
     {
+        clog(VerbosityTrace, "p2pcap") << p2pPacketTypeToString(DisconnectPacket) << " from " << m_info.id << "@" << m_socket->remoteEndpoint();
```

Comment:
Looks like this can be moved out of `switch` and done once for any packet

Reply:
Ah of course